tools/wpt/tests/test_wpt.py

```diff
@@ -188,6 +188,30 @@ def test_run_firefox(manifest_dir):
         del os.environ["MOZ_HEADLESS"]
 
 
+@pytest.mark.slow
+@pytest.mark.remote_network
+@pytest.mark.xfail(sys.platform == "win32",
+                   reason="Tests currently don't work on Windows for path reasons")
+def test_run_firefox_binary_args(manifest_dir):
```

Comment:
I think this test is pretty heavyweight for what it does. Can we fold it into another test, or just add an assert to the code?